std/regex/internal/parser.d

```diff
@@ -924,6 +924,7 @@ struct Parser(R)
                     addWithFlags(set, last, re_flags);
                     break L_CharTermLoop;
                 default:
+                    state = State.Char;
```

Comment:
Not if we come from `case State.PotentialTwinSymbolOperator`, which is why I explicitly set it now.